reports/Kentucky.md

```diff
@@ -34,6 +34,14 @@ A young woman was injured by a rubber bullet she took to the head.
 * [Video](https://twitter.com/shannynsharyse/status/1267015577266249728)
 * [Photo of Victim](https://twitter.com/shannynsharyse/status/1266631722239766528)
 
+### Police shoot at cars in traffic from overpass | May 31st
+
+Several police officers shoot from an overpass at cars stuck in traffic. Bystander reports at least one car window broken.
```

Comment:
Are we able to be specific about what they shot? Gas canisters? Less-lethal bullets?

Reply:
I'm going to merge this as-is now, but feel free to raise another PR if you want to address this.